.github/workflows/netbench.yml

```diff
@@ -186,6 +186,12 @@ jobs:
           sudo chmod +x bpftrace
           sudo mv bpftrace /usr/bin/
 
+      - name: Setup environment
+        run: |
+          # set larger socket buffers
+          sudo sysctl -w net.core.wmem_default=2000000
+          sudo sysctl -w net.core.rmem_default=2000000
```

Comment:
why did you add this? Maybe mention it in the PR description

Reply:
Added to the `Callouts` in the description